web/src/nix-search.ts

```diff
@@ -70,9 +70,11 @@ export class NixSearch extends LitElement {
       >
       </sl-input>
       ${this.loading
-        ? html`<sl-spinner
-            style="font-size: 50px; --track-width: 10px;"
-          ></sl-spinner>`
+        ? html` <div style="margin-top: 3em; width: 100%; text-align: center;">
```

Comment:
I prefer to keep the styles in the static style css block. Since styles are scoped to the component, we can select directly the `div` and `sl-spinner` elements:
```css
div {
  margin-top: 3em; 
  width: 100%; 
  text-align: center;
}
```